File: Api/Dtos/PostToList.cs
using System;
using Api.Models;

namespace Api.Dtos
{
    public class PostToList
    {
        public int PostId { get; set; }
        public string Content { get; set; }
        public DateTime TimePost { get; set; }
        public UserToList User { get; set; }
        public GamesToList Game { get; set; }
        public string ImageUrl { get; set; }
        public int LikeCount { get; set; }
        public int CommentCount { get; set; }
        public bool isLike { get; set; }
    }
}

File: Client/src/app/Models/PostToList.ts
import { User } from './User';
import { Game } from './Game';

export interface PostToList {
    postId: number,
    content: string,
    timePost: Date,
    user: User,
    game: Game,
    imageUrl: string,
    likeCount: number,
    commentCount: number,
    isLike: boolean
}